tests/integration/defs/accuracy/mixed_modality.py

```
@@ -0,0 +1,1165 @@
+            break
+    return samples
+
+
+def _iter_mmmu_rows(dataset_path: Path) -> Iterable[tuple[dict[str, Any], str]]:
+    """Yield MMMU rows across available configs and evaluation splits."""
+    from datasets import get_dataset_config_names, load_dataset
+
+    config_names: list[str] = []
+    try:
+        config_names = list(get_dataset_config_names(str(dataset_path)))
+    except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)